package.json

```diff
@@ -96,7 +97,7 @@
     "gulp-size": "^2.1.0",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-useref": "3.1.5",
+    "gulp-useref": "^4.0.0",
```

Comment:
I had some reservations about updating to this version, because it seems like `gulp-useref` is still relying on versions of `event-stream` that may be malicious:

https://github.com/jonkemp/gulp-useref/issues/256

Supposedly it's fine, but it still seemed disconcerting to me

Reply:
The latest `event-stream 4.0.1` no longer has malicious dependency `flatmap-stream` and should be safe, and `npm audit` doesn't report any errors.

Reply:
OK, sounds good